flask_static_digest/__init__.py

```diff
@@ -55,10 +54,19 @@ def _load_manifest(self, endpoint, scaffold):
         except (FileNotFoundError, Exception):
             pass
 
-    def _prepend_host_url(self, host, filename):
-        return urljoin(
-            self.host_url, "/".join([self.static_url_path, filename])
-        )
+    def _custom_url_join(self, url):
+        """
+        This function will join the host URL with another URL.
```

Comment:
Can you please change this entire description to:

```
Join the host URL with another URL path.
```

Reply:
Done